pex/bin/pex.py

```diff
@@ -470,18 +486,23 @@ def configure_clp_pex_environment(parser):
         default=[],
         type="str",
         action="append",
-        help="Constrain the selected Python interpreter. Specify with Requirement-style syntax, "
-        'e.g. "CPython>=2.7,<3" (A CPython interpreter with version >=2.7 AND version <3) '
-        'or "PyPy" (A pypy interpreter of any version). This argument may be repeated multiple '
-        "times to OR the constraints.",
+        help=(
+            "Constrain the selected Python interpreter. Specify with Requirement-style syntax, "
+            'e.g. "CPython>=2.7,<3" (A CPython interpreter with version >=2.7 AND version <3) '
+            'or "PyPy" (A pypy interpreter of any version). This argument may be repeated multiple '
+            "times to OR the constraints."
+        ),
     )
 
     group.add_option(
         "--rcfile",
         dest="rc_file",
         default=None,
-        help="An additional path to a pexrc file to read during configuration parsing. "
-        "Used primarily for testing.",
+        help=(
```

Comment:
Reworded to emphasize this appends, rather than replaces. And to mention the env var.